src/main/java/fruit/shop/service/StringConnector.java

```diff
@@ -0,0 +1,8 @@
+package fruit.shop.service;
+
+import java.util.Map;
+
+public interface StringConnector {
+    String getStringReport(Map<String, Integer> records);
+

```

Comment:
remove the last empty line in all classes/interfaces/methods where it is present
```suggestion
```